routes/articles.js

```diff
@@ -1,18 +1,28 @@
 'use strict';
 
 const express = require('express');
-const router = express();
-const bodyParser = require('body-parser');
-const jsonParser = bodyParser.json();
-const articleRoute = require('../database/articles.js');
-const PORT = 8000;
+const router = express.Router();
+const articleRoute = require('../database/articles');
 
-router.post('/', (req, res) => {
+router
+  .route('/')
+  .get((req, res) => {
+  res.render('templates/articles/index', articleRoute.getAllArticles());
```

Comment:
Indentation is weird in this file.

Reply:
tried to adjust indentation to appear like:

database.route('/')
  .get((req, res) => {
     code...})

however prettier adjusts the indentation back to how it currently is. I don't know if this is because of a setting for prettier but I am currently unable to adjust this.